crates/ruma-state-res/src/event_auth.rs

```diff
@@ -753,14 +836,14 @@ fn can_send_event(event: impl Event, ple: Option<impl Event>, user_level: Int) -
     true
 }
 
-/// Confirm that the event sender has the required power levels.
+/// Check if the given power level event is authorized.
 fn check_power_levels(
     room_version: &RoomVersion,
-    power_event: impl Event,
-    previous_power_event: Option<impl Event>,
-    user_level: Int,
+    power_level_event: impl Event,
+    previous_power_level_event: Option<impl Event>,
```

Comment:
Should be called `power_levels_event` (extra `s`) probably?

Reply:
Yes it should!

I actually fixed that in another branch that will be in a follow-up more invasive PR where I changed the name of the variables again… 

So maybe I should revert those changes here.

Reply:
Alright I reverted some variable name changes.